README.md

```diff
@@ -18,6 +18,11 @@ Web Components are a new browser feature that provides a standard component mode
 - CSS changes
     - The CSS WG works on [CSS Scoping](https://drafts.csswg.org/css-scoping/) and [CSS Shadow Parts](https://drafts.csswg.org/css-shadow-parts/), which help dealing with shadow trees with various selectors. Various other parts of CSS and its object model are also impacted by shadow trees and directly worked on in the various CSS specificaions.
     - [Issues against the CSS WG repository](https://github.com/w3c/csswg-drafts/labels/topic%3A%20shadow).
+- HTML Modules
+    - Successor to the abandoned *[HTML Imports](https://w3c.github.io/webcomponents/spec/imports/)*, allows HTML markup to be requested by a component. Spec work is being incubated upstream in whatwg/html (see [PR](todo)). **Note: PR creation is still in progress; replace with actual URL when opened.** For new issues, please file against whatwg/html or here in w3c/webcomponents. See also the [HTML Modules explainer](proposals/html-modules-explainer.md), [initial proposal](proposals/html-modules-proposal.md), and [earlier design ideas](proposals/HTML-Imports-and-ES-Modules.md).  
```

Comment:
I suspect `[PR](todo)` will create a 404. Maybe better to leave that out for now?

Reply:
Yep, I won't merge till this is updated with real links :)

Reply:
Real link is in.